sartography/spiff-arena
# Feature/restrict call activity processes

This adds restrictions to what processes can be used as call activities. The user must have access to run the process model in order to use it as a call activity. Otherwise it will not be returned in the processes list and will error if a user attempts to save a bpmn diagram that is using the process identifier.

To Test:
1 - ensure a process that the user cannot run is not returned in the processes list when trying to set a called element in a bpmn diagram
2 - If copying and pasting the process id that the user does not have access into the called element text box and then attempting to save, an error is given
